lib/livebook_web/live/settings_live/env_var_component.ex

```diff
@@ -0,0 +1,101 @@
+defmodule LivebookWeb.SettingsLive.EnvVarComponent do
+  use LivebookWeb, :live_component
+
+  alias Livebook.Settings
+  alias Livebook.Settings.EnvVar
+
+  @impl true
+  def mount(socket) do
+    env_var = %EnvVar{}
+    operation = :new
+    changeset = Settings.change_env_var(env_var)
+
+    {:ok, assign(socket, changeset: changeset, env_var: env_var, operation: operation)}
+  end
```

Comment:
```suggestion
```

The first `update/2` is called immediately after the mount and it already sets all of these :)